src/pyOpenMS/CMakeLists.txt

```diff
@@ -616,7 +615,7 @@ if ("${PY_NUM_MODULES}" GREATER "1")
         LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/pyOpenMS/pyopenms"
         RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/pyOpenMS/pyopenms"
       )
-      #add_dependencies(pyopenms_${nr} compile_pxds) # probably not necessary since the dependency is given through the files
+      add_dependencies(pyopenms_${nr} compile_pxds) # this is necessary for make (not ninja), to avoid race conditions
```

Comment:
Why?

Reply:
cause make is stupid and every pyopenms_X target tries to call this dependency, instead of building it before the parallel branch